code/modules/mining/mine_items.dm

```diff
@@ -287,7 +287,7 @@ proc/move_mining_shuttle()
 		var/turf/target = get_turf(A)
```

Comment:
There's a generic fire proc that covers all this bullshit, why is it not being used?

Reply:
Show me the way sensei

Reply:
the deed is done